spearhead-blocks/parts/footer.html

```diff
@@ -0,0 +1,11 @@
+<!-- wp:group {"layout":{"inherit":true}} -->
+<div class="wp-block-group">
+	<!-- wp:group {"style":{"spacing":{"padding":{"top":"80px","bottom":"30px"}}}} -->
+	<div class="wp-block-group" style="padding-top:80px;padding-bottom:30px">
+		<!-- wp:paragraph {"align":"center"} -->
+		<p class="has-text-align-center">Proudly Powered by <a href="https://wordpress.org" rel="nofollow">WordPress</a></p>
```

Comment:
The original theme includes the site title in the footer, next to the 'powered by WordPress' line:

<img width="447" alt="image" src="https://user-images.githubusercontent.com/1645628/187716796-c8387860-de7d-4caf-bf1a-e7b6a22b1227.png">